code/modules/defenses/sentry.dm

```diff
@@ -22,10 +25,28 @@
 	var/omni_directional = FALSE
 	var/sentry_range = SENTRY_RANGE
 
+	has_camera = TRUE
+
 	var/damage_mult = 1
 	var/accuracy_mult = 1
+	var/burst = 1
 	handheld_type = /obj/item/defenses/handheld/sentry
 
+	var/enaged_timer = null // timer triggered when sentry gun shoots at a target to not spam the laptop
```

Comment:
enaged - these could also potentially be COOLDOWN_DECLAREs if you care to change it